code/game/atoms_movable.dm

```diff
@@ -13,15 +13,23 @@
 	var/verb_yell = "yells"
 	var/inertia_dir = 0
 	var/pass_flags = 0
-	glide_size = 8
 
+	//pixel movement shizwa
+	glide_size = 0
+	animate_movement = 0
+	bound_x = 0
+	bound_y = 0
+	bound_width = 32

```

Comment:
An SS13 human is (at most, South direction) 17px wide (round up to 18? down to 16?), this might alleviate some of the "I'm stuck on a door", "Woops, wall in the way", "FUCKING DOORS" issues.

Also think about having the East/West dirs in move change the bounds (I think byond allows that without things getting jittery) width from 17 to 10.